### PR TITLE
fix old import/export avm docs

### PR DESCRIPTION
**Pull request by ceyonur**
we removed import,export AVAX from avm recently. PR fixes some old docs mentioning these APIs. Also fixes some invalid signatures for import/export apis in general.

### CODE REVIEW

**Comment by yulin-dong** [comment]
After you merge this PR, can you also duplicate the changes in https://github.com/ava-labs/avax-docs? Sorry for the duplicate work for now.

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 1]
```diff
@@ -517,7 +517,9 @@ avax.import({
 
 * `to` is the address the asset is sent to. This must be the same as the `to` argument in the corresponding call to the C-Chain's `export`.
 * `sourceChain` is the ID or alias of the chain the asset is being imported from. To import funds from the X-Chain, use `"X"`.
-* `username` is the user that controls `to`.
+* `baseFee` is the base fee that should be used when creating the transaction. If ommitted, a suggested fee will be used.
```
Comment (danlaine) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:520]:
```suggestion
* `baseFee` is the base fee that should be used when creating the transaction. If omitted, a suggested fee will be used.
```

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 2]
```diff
@@ -500,14 +499,15 @@ This gives response:
...
-Finalize the transfer of a non-AVAX or AVAX from the X-Chain to the C-Chain. Before this method is called, you must call the X-Chain's [`avm.export`](exchange-chain-x-chain-api.md#avm-export) method to initiate the transfer.
+Finalize the transfer of a non-AVAX or AVAX from the X-Chain to the C-Chain. Before this method is called, you must call the X-Chain's [`avm.export`](exchange-chain-x-chain-api.md#avm-export) method with assetID `AVAX` to initiate the transfer.
 
 #### Signature
 
 ```go
 avax.import({
     to: string,
     sourceChain: string,
+    baseFee: int, \\ optional
```
Comment (danlaine) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:510]:
```suggestion
    baseFee: int, // optional
```

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 3]
```diff
@@ -555,14 +557,15 @@ curl -X POST --data '{
...
-Finalize a transfer of AVAX from the X-Chain to the C-Chain. Before this method is called, you must call the X-Chain's [`avm.exportAVAX`](exchange-chain-x-chain-api.md#avm-exportavax) method to initiate the transfer.
+Finalize a transfer of AVAX from the X-Chain to the C-Chain. Before this method is called, you must call the X-Chain's [`avm.export`](exchange-chain-x-chain-api.md#avm-export) method with assetID `AVAX` to initiate the transfer.
 
 #### Signature
 
 ```go
 avax.importAVAX({
     to: string,
     sourceChain: string,
+    baseFee: int, \\ optional
```
Comment (danlaine) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:568]:
```suggestion
    baseFee: int, // optional
```

File: build/avalanchego-apis/contract-chain-c-chain-api.md [hunk 4]
```diff
@@ -572,7 +575,9 @@ avax.importAVAX({
 
 * `to` is the address the AVAX is sent to. It should be in hex format.
 * `sourceChain` is the ID or alias of the chain the AVAX is being imported from. To import funds from the X-Chain, use `"X"`.
-* `username` is the user that controls `to`.
+* `baseFee` is the base fee that should be used when creating the transaction. If ommitted, a suggested fee will be used.
```
Comment (danlaine) [inline build/avalanchego-apis/contract-chain-c-chain-api.md:578]:
```suggestion
* `baseFee` is the base fee that should be used when creating the transaction. If omitted, a suggested fee will be used.
```

File: build/avalanchego-apis/exchange-chain-x-chain-api.md [hunk 5]
```diff
@@ -581,7 +581,7 @@ curl -X POST --data '{
 
 ### avm.export
 
-Send a non-AVAX from the X-Chain to the P-Chain or C-Chain. After calling this method, you must call [`avax.import`](contract-chain-c-chain-api.md#avax-import) on the C-Chain to complete the transfer.
+Send an asset from the X-Chain to the P-Chain or C-Chain. After calling this method, you must call [C-Chain's `avax.import`](contract-chain-c-chain-api.md#avax-import) or [P-Chain's `avax.importAVAX`](platform-chain-p-chain-api.md#avax-importAVAX) to complete the transfer.
```
Comment (danlaine) [inline build/avalanchego-apis/exchange-chain-x-chain-api.md:584]:
```suggestion
Send an asset from the X-Chain to the P-Chain or C-Chain. After calling this method, you must call the [C-Chain's `avax.import`](contract-chain-c-chain-api.md#avax-import) or the [P-Chain's `avax.importAVAX`](platform-chain-p-chain-api.md#avax-importAVAX) to complete the transfer.
```

File: build/avalanchego-apis/exchange-chain-x-chain-api.md [hunk 6]
```diff
@@ -619,7 +621,24 @@ curl -X POST --data '{
     "method" :"avm.export",
     "params" :{
         "to":"C-avax1q9c6ltuxpsqz7ul8j0h0d0ha439qt70sr3x2m0",
-        "amount": 500,
+        "amount": 10,
```
Comment (danlaine) [inline build/avalanchego-apis/exchange-chain-x-chain-api.md:624]: Did you mean to put 2 curl calls here?

File: build/avalanchego-apis/exchange-chain-x-chain-api.md [hunk 7]
```diff
@@ -1140,7 +1159,7 @@ This gives response:
 
 ### avm.import
 
-Finalize a transfer of AVAX from the P-Chain or C-Chain to the X-Chain. Before this method is called, you must call the P-Chain’s [`platform.exportAVAX`](platform-chain-p-chain-api.md#platform-exportavax) or C-Chain’s [`avax.export`](contract-chain-c-chain-api.md#avax-export) method to initiate the transfer.
+Finalize a transfer of asset from the P-Chain or C-Chain to the X-Chain. Before this method is called, you must call the P-Chain’s [`platform.exportAVAX`](platform-chain-p-chain-api.md#platform-exportavax) or C-Chain’s [`avax.export`](contract-chain-c-chain-api.md#avax-export) method to initiate the transfer.
```
Comment (danlaine) [inline build/avalanchego-apis/exchange-chain-x-chain-api.md:1162]:
```suggestion
Finalize a transfer of an asset from the P-Chain or C-Chain to the X-Chain. Before this method is called, you must call the P-Chain’s [`platform.exportAVAX`](platform-chain-p-chain-api.md#platform-exportavax) or C-Chain’s [`avax.export`](contract-chain-c-chain-api.md#avax-export) method to initiate the transfer.
```

File: build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md [hunk 8]
```diff
@@ -141,20 +141,20 @@ Now we have everything we need to transfer the tokens.
...
-        "to":"C-avax1jggdngzc9l87rgurmfu0z0n0v4mxlqta0h3k6e",   
-        "destinationChain": "C",    
-        "amount": 5000000,  
-        "username":"myUsername",    
-        "password":"myPassword" 
-    }   
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"avm.export`",
```
Comment (danlaine) [inline build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md:150]:
```suggestion
    "method" :"avm.export",
```

File: build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md [hunk 9]
```diff
@@ -141,20 +141,20 @@ Now we have everything we need to transfer the tokens.
...
+    "params" :{
+        "destinationChain": "C",
```
Comment (danlaine) [inline build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md:152]: Is this correct? I don't think `ExportArgs` has a `destinationChain` field

File: build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md [hunk 10]
```diff
@@ -163,14 +163,14 @@ Since your keystore user owns the corresponding private key on the C-Chain, you
...
-    "id"     :1,    
-    "method" :"avax.importAVAX",    
-    "params" :{ 
-        "to":"0x4b879aff6b3d24352Ac1985c1F45BA4c3493A398",  
-        "sourceChain":"X",  
-        "username":"myUsername",    
-        "password":"myPassword" 
-    }   
+    "id"     :1,
+    "method" :"avax.importAVAX",
```
Comment (danlaine) [inline build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md:167]: Can we pls replace this with `avax.import`? `avax.importAVAX` is deprecated

File: build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md [hunk 11]
```diff
@@ -197,16 +197,16 @@ Once your AVAX has been transferred to the C-Chain, you can use it to deploy and
 Now, you can move AVAX back from the C-Chain to the X-Chain. First we need to export:
 
 ```cpp
-curl -X POST --data '{  
-    "jsonrpc":"2.0",    
-    "id"     :1,    
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
     "method" :"avax.exportAVAX",
```
Comment (danlaine) [inline build/tutorials/platform/transfer-avax-between-x-chain-and-c-chain.md:203]: Can we pls replace this with `avax.export`? `avax.exportAVAX` is deprecated

File: build/tutorials/platform/transfer-avax-between-x-chain-and-p-chain.md [hunk 12]
```diff
@@ -77,18 +77,19 @@ As you may have noticed while transferring AVAX using the Avalanche Wallet, a cr
...
 
 ```cpp
 curl -X POST --data '{
     "jsonrpc":"2.0",
     "id"     :1,
-    "method" :"avm.exportAVAX",
+    "method" :"avm.export",
     "params" :{
         "to":"P-avax1wkmfja9ve3lt3n9ye4qp3l3gj9k2mz7ep45j7q",
         "destinationChain": "P",
```
Comment (danlaine) [inline build/tutorials/platform/transfer-avax-between-x-chain-and-p-chain.md:91]: I don't think `avm.export` has a `destinationChain` argument

**Review by danlaine — CHANGES_REQUESTED** [review]

File: build/avalanchego-apis/exchange-chain-x-chain-api.md (hunk 6 above)
Comment (ceyonur) [reply build/avalanchego-apis/exchange-chain-x-chain-api.md:624]: yeah but it seems weird, just removed it.

**Review by danlaine — APPROVED** [review]

**Comment by danlaine** [comment]
@ceyonur ty ser

**Pull request merged**